packages/ketcher-macromolecules/src/components/monomerLibrary/RnaBuilder/RnaEditor/RnaEditorExpanded/RnaEditorExpanded.tsx

```diff
@@ -146,12 +146,18 @@ export const RnaEditorExpanded = ({
   const onChangeName = (event: ChangeEvent<HTMLInputElement>) => {
     if (isEditMode) {
       const newPresetName = event.target.value;
-      if (newPresetName.trim() !== '') {
+      if (newPresetName !== '') {
         setNewPrest({
           ...newPreset,
           name: newPresetName.trim(),
           editedName: true,
         });
+      } else {
+        setNewPrest({
+          ...newPreset,
+          name: '',
+          editedName: true,
+        });
       }
```

Comment:
How about we just remove this judgement `if (newPresetName !== '')` and setNewPrest directly?

Reply:
@StarlaStarla Yes, we can simplify it that way. However, it will always trim names, removing leading/trailing spaces even on empty input. If we're okay with this, I'll update the code.

Reply:
I would think it is ok. But maybe we could consult Roman on this.